createNativeStackNavigator/README.md

```diff
@@ -185,6 +185,10 @@ A Boolean to that lets you opt out of insetting the header. You may want to * se
 
 Boolean indicating whether the navigation bar is translucent.
 
+#### `onTransitionProgress`
+
+A callback called every frame during the transition of screens.
```

Comment:
Maybe also explain that the event provides the information of the transition progress (0-1)